io.openems.edge.core/src/io/openems/edge/app/common/props/CommonProps.java

```diff
@@ -109,8 +108,11 @@ PARAM extends BundleParameter> AppDef<APP, PROP, PARAM> installationHintOfUnoffi
 				(app, property, l, parameter) -> TranslationUtil.getTranslation(parameter.bundle,
 						"unofficialAppWarning.text1"), //
 				(app, property, l, parameter) -> TranslationUtil.getTranslation(parameter.bundle,
-						"unofficialAppWarning.text2")),
-				def -> def.setAutoGenerateField(!OpenemsConstants.MANUFACTURER.equals("OpenEMS Association e.V.")));
+						"unofficialAppWarning.text2"))
+		// TODO ,def ->
+		// def.setAutoGenerateField(!OpenemsConstants.MANUFACTURER.equals("OpenEMS
+		// Association e.V."))
+		);
```

Comment:
@michaelgrill I need your support here, how to solve this.

Reply:
As i already talked with you about this in general handling apps, @clehne may also have an opinion on this. We were discussing to move the apps maybe completely to the OEM specific bundle then everyone has their own "available" apps, but it would probably result in a lot of duplicated code when not sharing a simple app like "KEBA EVCS" or so.

For now you can just remove those lines 🙂

Reply:
Thanks Michael. As also Christians feedback was positive, I'll continue with this PR to have a first OEM bundle. From there we'll discuss individually how we extract the remaining OEM specific changes like AggregatedInflux, Apps, etc.

Reply:
> As i already talked with you about this in general handling apps, @clehne may also have an opinion on this. We were discussing to move the apps maybe completely to the OEM specific bundle then everyone has their own "available" apps, but it would probably result in a lot of duplicated code when not sharing a simple app like "KEBA EVCS" or so.
> 
> For now you can just remove those lines 🙂

@michaelgrill Right now I do not have a final opinion on apps handling. I think also that we should finish this PR and delay the app handling to a new PR. 

In general it would be good to 

- have a common base for most of the apps. 
- have a OEM specific link for each app url and probably for the app image.
- have the opportunity for own apps. 

The keba EVCS app is a good example. We tried to use it but this leads to problems with our evcs cluster app for charge parks. Later we needed a lot of workarounds for e.g. alfen, compleo apps just because of the keba evcs app.
In the end we  decided to put our own different keba app beside the official keba app. 
I think this is a good compromise for now, as your and our company proceed with different speeds. But later we should find a way to integrate both sets of apps into one base.